rewrite-gradle/src/main/java/org/openrewrite/gradle/trait/Traits.java

```diff
@@ -23,6 +23,10 @@ public static GradleDependency.Matcher gradleDependency() {
         return new GradleDependency.Matcher();
     }
 
+    public static SpringDependencyManagementPluginEntry.Matcher springDependencyManagementPluginEntry() {
+        return new SpringDependencyManagementPluginEntry.Matcher();
+    }
```

Comment:
As @timtebeek confirmed for me, we're no longer going to use the factory pattern going forward, so we can drop this method.